pdns/ueberbackend.cc

```diff
@@ -273,16 +281,35 @@ void UeberBackend::reload()
   }
 }
 
+void UeberBackend::updateDomainCache() {
+  if (!g_domainCache.isEnabled()) {
+    return;
+  }
+
+  vector<tuple<DNSName, int>> domain_indices;
+
+  for (vector<DNSBackend*>::iterator i = backends.begin(); i != backends.end(); ++i )
+  {
+    vector<DomainInfo> domains;
+    (*i)->getAllDomains(&domains, false);
+    for(const auto& di: domains) {
+      domain_indices.push_back({di.zone, (int)di.id});  // this cast should not be necessary
```

Comment:
If we make `di` non-const we can move `di.zone` here.

Reply:
done